src/Altinn.Platform/Altinn.Platform.Events/Events/Services/EventsService.cs

```diff
@@ -17,21 +25,36 @@ namespace Altinn.Platform.Events.Services
     public class EventsService : IEventsService
     {
         private readonly IPostgresRepository _repository;
+        private readonly IQueueService _queue;
+        private readonly ILogger<IEventsService> _logger;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="EventsService"/> class.
         /// </summary>
-        public EventsService(IPostgresRepository repository)
+        public EventsService(
+            IPostgresRepository repository,
+            IQueueService queue,
+            ILogger<IEventsService> logger)
         {
             _repository = repository;
+            _queue = queue;
+            _logger = logger;
         }
 
         /// <inheritdoc/>
         public async Task<string> StoreCloudEvent(CloudEvent cloudEvent)
         {
             cloudEvent.Id = Guid.NewGuid().ToString();
             cloudEvent.Time = null;
-            return await _repository.Create(cloudEvent);
+            string cloudEventId = await _repository.Create(cloudEvent);
+            PushQueueReceipt receipt = await _queue.PushToQueue(JsonSerializer.Serialize(cloudEvent));
+
+            if (!receipt.Success)
+            {
+                _logger.LogError("// EventsService // StoreCloudEvent // Failed to push event {EventId} to queue. Exception {Exception}", cloudEventId, receipt.Exception);
+            }
+
```

Comment:
We are logging the eventId that failed, so by monitoring the logs we will know which events need to be manually pushed.. Not sure how this will be fixed, but I think the first step is to ensure that we know which events haven't been pushed